boards/arm/c5471/c5471evm/configs/httpd/defconfig

```diff
@@ -35,7 +35,6 @@ CONFIG_RAM_SIZE=285212672
 CONFIG_RAM_START=0
 CONFIG_RR_INTERVAL=200
 CONFIG_SCHED_HPWORK=y
-CONFIG_SCHED_LPWORK=y
```

Comment:
And why `CONFIG_SCHED_LPWORK=y` is removed from `defconfig` files? I see
```
#  if defined(CONFIG_SCHED_LPWORK) && defined(CONFIG_SCHED_HPWORK)
#    define LPWORK (HPWORK+1) /* Low priority, kernel-mode work queue */
#  else
#    define LPWORK HPWORK     /* Redirect low-priority references */
#  endif
```
so if both `CONFIG_SCHED_HPWORK` and `CONFIG_SCHED_LPWORK` are `y` we are having separate `LPWORK` and `HPWORK` otherwise `LPWORK` is redirected to the `HPWORK`. Does this change mean that all `LPWORK`'s will start run in high priority?

Reply:
CONFIG_SCHED_LPWORK is added by PR https://github.com/apache/incubator-nuttx/pull/6330, because that patch let NET_TCP depend on SCHED_LPWORK. Since this patch change NET_TCP to depend on SCHED_WORKQUEUE, we can revert all related change from #6330.

Reply:
I see. Thanks!